SherpaBackEnd/Services/TeamMemberService.cs

```diff
@@ -0,0 +1,40 @@
+using SherpaBackEnd.Dtos;
+using SherpaBackEnd.Exceptions;
+using SherpaBackEnd.Model;
+
+namespace SherpaBackEnd.Services;
+
+public class TeamMemberService : ITeamMemberService
+{
+    private readonly ITeamRepository _inMemoryTeamRepository;
+
+    public TeamMemberService(ITeamRepository inMemoryTeamRepository)
+    {
+        _inMemoryTeamRepository = inMemoryTeamRepository;
+    }
+
+    public async Task AddTeamMemberToTeamAsync(AddTeamMemberDto addTeamMemberDto)
+    {
+        try
+        {
+            await _inMemoryTeamRepository.AddTeamMemberToTeamAsync(addTeamMemberDto.TeamId, addTeamMemberDto.TeamMember);
+        }
+        catch (Exception error)
+        {
+            throw new ConnectionToRepositoryUnsuccessfulException(error.Message, error);
```

Comment:
This is strange. Any error of tha repo is because of the connection? I don't think this should be like that. If there is a unhandled exception is should continue as it is